hack/observability/alloy/values.yaml

```diff
@@ -1,30 +1,50 @@
-# Configuration for alloy - https://github.com/grafana/alloy/blob/main/operations/helm/charts/alloy/values.yaml
-config:
-  # publish data to loki
-  clients:
-    - url: http://loki:3100/loki/api/v1/push
-      # We have to set the tenant_id/OrgID to push logs to Loki
-      # otherwise we get an auth error.
-      tenant_id: 1
+# # Configuration for alloy - https://github.com/grafana/alloy/blob/main/operations/helm/charts/alloy/values.yaml
+# config:
+#   # publish data to loki
+#   clients:
+#     - url: http://loki:3100/loki/api/v1/push
+#       # We have to set the tenant_id/OrgID to push logs to Loki
+#       # otherwise we get an auth error.
+#       tenant_id: 1
 
-  snippets:
-    pipelineStages:
-      # Parse cluster and machine to make them available as labels.
-      - cri: {}
-      - json:
-          expressions:
-            controller:
-            Cluster: join('/',[Cluster.namespace,Cluster.name])
-            Machine: join('/',[Machine.namespace,Machine.name])
-            KubeadmControlPlane: join('/',[KubeadmControlPlane.namespace,KubeadmControlPlane.name])
-            MachineDeployment: join('/',[MachineDeployment.namespace,MachineDeployment.name])
-            MachineSet: join('/',[MachineSet.namespace,MachineSet.name])
-            MachinePool: join('/',[MachinePool.namespace,MachinePool.name])
-      - labels:
-          controller:
-          Cluster:
-          Machine:
-          KubeadmControlPlane:
-          MachineDeployment:
-          MachineSet:
-          MachinePool:
+#   snippets:
+#     pipelineStages:
+#       # Parse cluster and machine to make them available as labels.
+#       - cri: {}
+#       - json:
+#           expressions:
+#             controller:
+#             Cluster: join('/',[Cluster.namespace,Cluster.name])
+#             Machine: join('/',[Machine.namespace,Machine.name])
+#             KubeadmControlPlane: join('/',[KubeadmControlPlane.namespace,KubeadmControlPlane.name])
+#             MachineDeployment: join('/',[MachineDeployment.namespace,MachineDeployment.name])
+#             MachineSet: join('/',[MachineSet.namespace,MachineSet.name])
+#             MachinePool: join('/',[MachinePool.namespace,MachinePool.name])
+#       - labels:
+#           controller:
+#           Cluster:
+#           Machine:
+#           KubeadmControlPlane:
+#           MachineDeployment:
+#           MachineSet:
+#           MachinePool:
```

Comment:
We want to reflect this in some way in the alloy config and drop all the commented parts here (also above).